msi/GitProduct.wxs

```diff
@@ -1,76 +1,170 @@
-<?xml version='1.0'?>
-
-<?if $(sys.BUILDARCH)=x64 ?>
-  <?define SixtyFourBit=64 ?>
-<?else?>
-  <?define SixtyFourBit ?>
-<?endif?>
-
-<Wix xmlns='http://wixtoolset.org/schemas/v4/wxs'
-     xmlns:swid='http://wixtoolset.org/schemas/v4/wxs/tag'>
+<?xml version='1.0'?>

```

Comment:
Yeah. Stupid VS or notepad++ or something.... I need to make sure these files have their line endings normalized to prevent this kind of shenanigans. Sorry about that.